storage/s3_sdk/list_gcs_buckets_test.go

```diff
@@ -17,29 +17,78 @@ package s3sdk
 
 import (
 	"bytes"
+	"context"
+	"fmt"
 	"io/ioutil"
 	"log"
 	"os"
 	"strings"
 	"testing"
+
+	"cloud.google.com/go/storage"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
 )
 
+var serviceAccountEmail = os.Getenv("GOLANG_SAMPLES_SERVICE_ACCOUNT_EMAIL")
+var storageClient *storage.Client
+
 func TestMain(m *testing.M) {
 	log.SetOutput(ioutil.Discard)
 	s := m.Run()
 	log.SetOutput(os.Stderr)
+
+	ctx := context.Background()
+	storageClient, _ = storage.NewClient(ctx)
+	defer storageClient.Close()
+
+	tc, _ := testutil.ContextMain(m)
+	key, err := createTestKey(tc.ProjectID)
```

Comment:
Does this need to be in `TestMain`? If there is only one test, we can create/delete the key in there.